src/controller.py

```diff
@@ -65,6 +65,11 @@ def run_command_and_get_its_response(self) -> dict:
         """Performs wanted action and returns details of what happened."""
         if self.command == "ping":
             return {"response": "pong"}
+        elif self.command == "log":
+            text = self.request_dict["text"]
+            to_log = f"New test case: {text}"
```

Comment:
I would omit the text and just log the `text`. Suite can add the `New test case: ` and it'll be able to use the `log` command for anything. Exactly as you write in the docs.